### PR TITLE
Updated Meetup demo to use Apollo hooks

### PR DESCRIPTION
**Pull request by Vultraz**
Still need to refactor out the use of the `withApollo` HOC but I'll do that separately.

### CODE REVIEW

**Comment by Vultraz** [comment]
@timleslie rebased

File: yarn.lock [hunk 1]
```diff
@@ -92,7 +92,7 @@
     ts-invariant "^0.4.4"
     tslib "^1.10.0"
 
-"@apollo/react-ssr@^3.1.3":
+"@apollo/react-ssr@3.1.3":
```
Comment (timleslie) [inline yarn.lock]: Is this the correct yarn.lock file? I don't understand why this line would have changed?

Comment (Vultraz) [reply yarn.lock]: My bad! Fixed.

**Comment by Vultraz** [comment]
@timleslie fixed another merge conflict

**Pull request merged**